src/plugins/intel_gpu/src/graph/graph_optimizer/update_loop_primitive_map.cpp

```diff
@@ -14,18 +15,27 @@ using namespace cldnn;
 
 void update_loop_primitive_map::run(program& p) {
```

Comment:
Now that it is not only updating loop, how about modify the function name to update_inner_program_io_map?

Reply:
Fixed it.